src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -498,6 +498,22 @@ abstract class PipelineTargets(config: Config) {
       config
     )
 
+    lazy private[overwatch] val verboseAuditTarget: PipelineTable = PipelineTable(
+      name = "verboseAuditTarget_gold",
```

Comment:
change this name to something friendlier...perhaps notebookCommands_gold (feel free to find something better)

certainly we don't want to put the word "audit" in a gold table -- customers may have misplaced expectations